pyproject.toml

```diff
@@ -179,10 +158,12 @@ exclude = '''
 
 [tool.ruff]
 exclude = [
+  "*.ipynb",
   ".bzr",
   ".direnv",
   ".eggs",
   ".git",
+  ".git",
```

Comment:
Very minor comment here -- duplication of ".git"

Reply:
Good spot, thanks :+1: resolved.